CHANGELOG.MD

```diff
@@ -1,3 +1,21 @@
+#### v2.1.0
+
+- performance testing (browserMob and TTFB timings)
+- improvements for clicking element, every step waits for visibilityOf element
```

Comment:
Maybe, it would be nice if we add here a note, which informs a user that `wait` before a step is no longer required, in most common cases.